### ISSUE TITLE
I can't execute catbots

### ISSUE THREAD

**Issue by clowcheck** [nullworks/cathook#1337]
Every time I execute your shitty code, It says please install cathook. I dunno how you guys program, when I look at  your issues you don't help people, you just block users, that proves you have zero knowledge about coding. Take my advise and cancel your project.

![unknown](https://user-images.githubusercontent.com/61922490/106656064-3c468680-65ab-11eb-97c7-97ff8cb74a29.png)

**Comment by TotallyNotElite** [comment]
This is hilarious, thanks for the laugh dude.

I don't know why you expect to receive help with THAT attitude, just do us all a favour and get out, will you?

"Zero knowledge about coding" lmaooo

If you had knowledge of coding, perhaps you could find out why that error is being shown, just a hint.

**Comment by clowcheck** [comment]
Are u mad? :)

**Comment by TotallyNotElite** [comment]
Nope, I'm getting a good laugh out of this

**Comment by clowcheck** [comment]
Same you create a project but you can't even help people, what a lame attitude :)

**Comment by TotallyNotElite** [comment]
I would've helped you if you didn't have such an attitude 😉

**Comment by clowcheck** [comment]
By the way, If you need this: https://www.wikihow.com/Get-a-Life    the website will be always open for you

**Comment by clowcheck** [comment]
You can't help people, but I can help people that's difference between us, I can help you with getting a life, take a fresh air, clearly you can't think health.

**Comment by akiy-33** [comment]
What? you would have helped someone if they didn't have such an attitude? by sending "look at our empty wiki page" or closing thread without any kind of such a great attitude? Cmon mate you guys could do a better job.

**Comment by TotallyNotElite** [comment]
Catbots should not be used by people who can't solve basic problems themselves. It really only takes a second of activating your brain to find out why this error occurred, but I'm not tellin' ya.

**Comment by clowcheck** [comment]
> Catbots should not be used by people who can't solve basic problems themselves. It really only takes a second of activating your brain to find out why this error occurred, but I'm not tellin' ya.

So, you couldn't active your brain, Is this your reason to block users.

**Comment by TotallyNotElite** [comment]
I blocked your other account because you have an attitude and I don't want to deal with you. Not my problem dude, should've kept your cool

**Comment by clowcheck** [comment]
Well, I was getting help, Is that problem for you?

**Comment by TotallyNotElite** [comment]
No, people ask for help all the time and we help them out. Give me one reason why I should even consider helping you.

**Comment by DavidStojko** [comment]
Fridge level iq 🤣 🤣

**Comment by clowcheck** [comment]
Of course, you are not even a coder, Why should I get help from you?  You can't help me even you want because you don't know how to fix the problem.

**Comment by clowcheck** [comment]
> Fridge level iq 🤣 🤣

Antisocial Gayymer iq :D

**Comment by clowcheck** [comment]
> Fridge level iq 🤣 🤣

Well, I'm not pasting my codes like you :D

**Comment by TotallyNotElite** [comment]
> Of course, you are not even a coder, Why should I get help from you? You can't help me even you want because you don't know how to fix the problem.

![image](https://user-images.githubusercontent.com/38938720/106658770-04851200-659e-11eb-8400-e88265fe2c67.png)

:fire::fire::fire::fire::fire::fire::fire::fire::fire::fire::fire::fire::fire::fire::fire::fire:

**Comment by clowcheck** [comment]
> > Of course, you are not even a coder, Why should I get help from you? You can't help me even you want because you don't know how to fix the problem.
> 
> ![image](https://user-images.githubusercontent.com/38938720/106658770-04851200-659e-11eb-8400-e88265fe2c67.png)
> 
> 🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥

Are you sure is this your account?

**Comment by TotallyNotElite** [comment]
You can check yourself :wink:

**Comment by DavidStojko** [comment]
> Of course, you are not even a coder, Why should I get help from you? You can't help me even you want because you don't know how to fix the problem.

Sorry didn't see you there totally incompetent person. I'm sure you will be able to fix your own issue with master coding skills you acquired by being mentally challenged. Reading a sentence is proving to be quite a hard task for you so i'l just assume its TNE's issue. How about you help by contributing to the code oh the great messiah of mentally challenged coders?

**Comment by clowcheck** [comment]
Imagine you are wasting your time on those useless projects, you better get a life.

**Comment by DavidStojko** [comment]
Nice contribution history oh great messiah of the mentally challenged coders 🤣 🤣

**Comment by clowcheck** [comment]
Well, mentally challenged codes got offended, You can't even solve problem instead of solving my problem, you are sending your antisocial gayyming projects, please stop decreasing my iq level, I need my iq.

**Comment by clowcheck** [comment]
We looked at your code and It looks like shit, are u guys trying to code yandere dev ?

**Comment by DavidStojko** [comment]
> odes got offended, You can't even solve problem instead of solving my problem, you are sending your antisocial gayyming projects, please stop decreasing my iq level, I need

You can't lose something you never even had in the first place. Otherwise you wouldn't be called the great messiah of mentally challenged coders. Now if you can excuse me I'm trying to find super duper important projects you're working on in your own github profile but its proving hard to find anything oh the great messiah of mentally challenged coders.

**Comment by clowcheck** [comment]
Wow mentally challenged coders are working super duper important projects(yandere dev sim.) Ok good luck on that :)

**Comment by DavidStojko** [comment]
You can't even type proper English yet you feel you're competent enough to trash talk other peoples code you wouldn't be able to write in a hundred of years?!?!?!? MAGNIFICENT!!!! It's exactly what i expected from the great messiah of the mentally challenged coders!!!!!!!

**Comment by TotallyNotElite** [comment]
Hold on let me get the popcorn :popcorn:

**Comment by clowcheck** [comment]
I have B2 English certificate on duolingo, I don't focus on you cuz I never care about grammar talking with anti socials so get a life please will ya ?

**Comment by DavidStojko** [comment]
MY DUDE FLEXING HIS DUOLINGO CERTIFICATE HE EARNED BY PLAYING MINIGAMES ON THE DAMN APP 🤣 🤣 
The great messiah of mentally challenged coders proved to be an even more interesting being than i originally anticipated!!!!!!!

**Comment by DavidStojko** [comment]
I'm still waiting on him to show us his super duper self pasted p2c he earns billions of dollars from 🤣 🤣

**Comment by clowcheck** [comment]
Well, you guys can continue to play your shitty games while I'm improving myself so I'm repeating again get a life :> I can see clearly that you have no job to do, shame :/

**Comment by clowcheck** [comment]
I have really doubt that you have great knowledge about the problem, you didn't fix my problem come on master USE YOUR BRAIN COME ON

**Comment by BenCat07** [comment]
ok so here is the answer to your issue: install cathook like it says in the readme on here. if you can't do that, then i'm sorry but you may want to learn how to read

**Comment by TotallyNotElite** [comment]
> I have really doubt that you have great knowledge about the problem, you didn't fix my problem come on master USE YOUR BRAIN COME ON

Can't fix what ain't broken mate

**Comment by BenCat07** [comment]
> ok so here is the answer to your issue: install cathook like it says in the readme on here. if you can't do that, then i'm sorry but you may want to learn how to read

this

**Comment by imgladnotu** [comment]
> ok so here is the answer to your issue: install cathook like it says in the readme on here. if you can't do that, then i'm sorry but you may want to learn how to read
 
 cant fix genuine brain issue

**Comment by DavidStojko** [comment]
> I have really doubt that you have great knowledge about the problem, you didn't fix my problem come on master USE YOUR BRAIN COME ON

Oh but you got a B2 certificate on duolingo I'm sure you can solve the problem yourself oh the great messiah of the mentally challenged coders.

**Comment by clowcheck** [comment]
> l cathook like it says in the

Well, I'll do it super brain, If this thing doesn't work I'll fuck you up again.

**Comment by DavidStojko** [comment]
OH NOESSSSS HES GOING TO SNITCH US TO HIS MOTHER

**Comment by imgladnotu** [comment]
> 
> 
> OH NOESSSSS HES GOING TO SNITCH US TO HIS MOTHER BECAUSE WE DONT WANT TO SPOON FEED HIM

dude bro im trembling

**Comment by clowcheck** [comment]
Well, can you get job in real life actually? I guess this explains a lot.

**Comment by clowcheck** [comment]
Well, It didn't work super coders, Did u copy and paste this code from somewhere else?

**Comment by clowcheck** [comment]
I think another user is having a problem, you better check #1337. issue :) maybe you super brains can help this guy :)

**Comment by BenCat07** [comment]
it works if you did what it said, but i guess you are unable to read

**Comment by TotallyNotElite** [comment]
@BenCat07 bro he has cathook on his desktop, that mean's it's installed!!!!

**Comment by DavidStojko** [comment]
> Well, can you get job in real life actually? I guess this explains a lot.

Why would I need a job I don't live in a 3rd world country like you 🤣 I can get by being a freelancer and still earn triple your monthly salary in week.

**Comment by DavidStojko** [comment]
> I think another user is having a problem, you better check #1338. issue :) maybe you super brains can help this guy :)

OMFG MY DUDE LINKED HIS OWN ISSUE AND THOUGHT ITS SOMEONE ELSE'S LMAOOOOO 🤣 🤣

**Comment by clowcheck** [comment]
Well, I'm from Norway so I'm not living a 3rd world country, maybe you guys can take a fresh air to open your dead brain cells. I think you are unqualified to find a job.

**Comment by clowcheck** [comment]
okay. then explain me how can i install cathook on my "MAIN TF2" ? your readme section isn't helping me about this issue.

**Comment by BenCat07** [comment]
install cathook and inject it atleast once into tf2

**Comment by DavidStojko** [comment]
> install cathook and inject it atleast once into tf2

Stop spoon feeding him and let him roast him more ples

**Comment by clowcheck** [comment]
Well, is too shame that u'll roast in your country without a job.

**Comment by clowcheck** [comment]
> install cathook and inject it atleast once into tf2

I already have done that

**Comment by BenCat07** [comment]
ls /opt/

if there is no cathook folder, no you have not

**Comment by DavidStojko** [comment]
simulink vibing so hard tho 🤣

**Comment by clowcheck** [comment]
> if there is no cathook folder, no you have not

I have cathook folder on my desktop.

**Comment by TotallyNotElite** [comment]
May I just ask what distro this is lmao

**Comment by clowcheck** [comment]
I'm currently using linux mint 20

**Comment by BenCat07** [comment]
> > if there is no cathook folder, no you have not
> 
> I have cathook folder on my desktop.

but is there one in /opt/

**Comment by BenCat07** [comment]
also gui shoupd be set to source mode in settings

**Comment by DavidStojko** [comment]
Slow down Light his pea brain can't understand all the words you said 🤣

**Comment by TotallyNotElite** [comment]
> I'm currently using linux mint 20

Dude, read the fucking wiki, is linux mint listed in the supported distros? Come on, we are all waiting for an answer

**Comment by BenCat07** [comment]
> also gui should be set to source mode in settings

you using mint probably made it fall back to binary modr instead which uses different paths

**Comment by BenCat07** [comment]
mint may not work, might do tho idk

**Comment by DavidStojko** [comment]
Yall are scaring him with all those tech speak.

**Comment by clowcheck** [comment]
> mint may not work, might do tho idk

You are the only one helps me about this problem, and TotallyNotElite have to gone he/she's not helping just insulting us, Thanks for everything.

**Comment by TotallyNotElite** [comment]
I'm hovering over that ban button so hard right now, you don't deserve a second of help. You did nothing but insult community members and even doubled down after being asked to calm the fuck down.

**Comment by DavidStojko** [comment]
> > Yall are scaring him with all those tech speak.
> 
> Get a life anti social gamer, I hope you'll get kicked from this project

Says a guy crying because no one wants to teach him how to read lmfao 🤣 🤣

**Comment by clowcheck** [comment]
> Yall are scaring him with all those tech speak.

Aaa wait I forgot that you were not a developer in this project

**Comment by imgladnotu** [comment]
> 
> 
> > Yall are scaring him with all those tech speak.
> 
> Aaa wait I forgot that you were not a developer in this project

who says you exactly need to be one. this is github. not some company working for a product lol

**Comment by TotallyNotElite** [comment]
Unlike you, @DavidStojko actually is capable of reading the "supported distros" text :laughing:. He's an active community member.

**Comment by DavidStojko** [comment]
Hes awfully quiet now 🤣

**Comment by clowcheck** [comment]
Well, you guys have problems, before fixing a user's problem fix your problems You I can arrange a psychiatric for you.

**Comment by clowcheck** [comment]
https://www.mind.org.uk/information-support/drugs-and-treatments/talking-therapy-and-counselling/how-to-find-a-therapist/

**Comment by DavidStojko** [comment]
> Well, you guys have problems, before fixing a user's problem fix your problems You I can arrange a psychiatric for you.

The only one that needs medical help here is you because you delusional as fuck 🤣 🤣 
Shizo tripping hes relevant lmao 🤣

**Comment by imgladnotu** [comment]
>Every time I execute your shitty code, It says please install cathook. I dunno how you guys program, when I look at your issues you don't help people, you just block users, that proves you have zero knowledge about coding. Take my advise and cancel your project.

__let that sink in__

**Comment by akiy-33** [comment]
Okay I solved his problem now. I was the only developer here, so the only one who can actually help is me. So yeah I would mark it as Closed now if they had better a sdk knowledge and provided a better code that wasn't an if else simulator but anyways guys. Thanks me later :salt: :salt: :salt: 😎

**Comment by clowcheck** [comment]
I think you are playing too much video games, you have to learn how to communicate with people, I guess you are result of bad parenting.

**Comment by BenCat07** [comment]
i blocked you btw.

**Comment by BenCat07** [comment]
have fun.

**Comment by imgladnotu** [comment]
lol, that's a 1

**Comment by akiy-33** [comment]
who cares anyways

**Comment by BenCat07** [comment]
i have alot of patience, but this is too far.

**Comment by imgladnotu** [comment]
:|

**Comment by TotallyNotElite** [comment]
@akiy-33 you are welcome to make a pull request if there is code you don't like. Us maintainers don't have time to look after code quality in older commits in a project of this size.

**Comment by akiy-33** [comment]
yeah sweats were dripping down your neck so that was a good shit

**Comment by akiy-33** [comment]
> @akiy-33 you are welcome to make a pull request if there is code you don't like. Us maintainers don't have time to look after code quality in older commits in a project of this size.

who are you lmao? i wasn't even talking to you?

**Comment by DavidStojko** [comment]
> I think you are playing too much video games, you have to learn how to communicate with people, I guess you are result of bad parenting.

Says a guy throwing a tantrum because we don't want to spoon feed him 🤣

**Comment by akiy-33** [comment]
learn to use operators retard

**Comment by TotallyNotElite** [comment]
Oh no a guy arguing on the internet, my feelings are being hurt! :snowflake::snowflake::snowflake:

Guys I am literally sweating right now!

**Comment by akiy-33** [comment]
anyways i am the guy who knows more programming here so you guys cannot stand a second next 2 m3

**Comment by TotallyNotElite** [comment]
> learn to use operators retard

Developer: doesn't put every condition on the exact same line
@akiy-33: l0l u can't code m8

**Comment by akiy-33** [comment]
> > learn to use operators retard
> 
> Developer: doesn't put every condition on the exact same line
> @akiy-33: l0l u can't code m8

can i see your "very very clean" code on this project? mate?

**Comment by TotallyNotElite** [comment]
@akiy-33 you don't have any public projects, there's no way for such a claim to be verified.

**Comment by akiy-33** [comment]
> @akiy-33 you don't have any public projects, there's no way for such a claim to be verified.

well you don't even have a private one so i pity on you

**Comment by akiy-33** [comment]
ughmmm where is your very very clean code on the cathook project?? commit link please?

**Comment by TotallyNotElite** [comment]
I'm an open source and free software developer, I publish all my code for everyone to use under copyleft licenses. You got a problem with that?

**Comment by akiy-33** [comment]
you aint a shit mate wu talking about stop claiming urself as a developer u wanker

**Comment by TotallyNotElite** [comment]
Yet I work on multiple projects with a reach of hundreds of thousands of monthly users. Curious, isn't it?

**Comment by akiy-33** [comment]
where is your very fucking clean code on cathook project? asking triple times

**Comment by akiy-33** [comment]
> Yet I work on multiple projects with a reach of hundreds of thousands of monthly users. Curious, isn't it?

i see no shit so do really i?

**Comment by imgladnotu** [comment]
> 
> 
> Yet I work on multiple projects with a reach of hundreds of thousands of monthly users. Curious, isn't it?

do we even have a solid grasp of the analytics on the claim?

**Comment by TotallyNotElite** [comment]
> where is your very fucking clean code on cathook project? asking triple times

I'm under no obligation to provide you with anything. Go search through the project commits yourself. If there's code you don't like, fix it, this is an open source project where contributions are welcome. But no, of course you won't do that because you're only here to claim your superiority :laughing:

A massive waste of my time, is what this is

**Comment by akiy-33** [comment]
> > where is your very fucking clean code on cathook project? asking triple times
> 
> I'm under no obligation to provide you with anything. Go search through the project commits yourself. If there's code you don't like, fix it, this is an open source project where contributions are welcome. But no, of course you won't do that because you're only here to claim your superiority 😆
> 
> A massive waste of my time, is what this is

okay if no claim then i claim your shitcode? so do you want?

**Comment by TotallyNotElite** [comment]
I'm going to lock this issue now, thanks for the entertainment, people pay real money for this shit, and we got it for free :dollar:

**Comment by Snephanie** [comment]
Yes please do

**Comment by akiy-33** [comment]
> I'm going to lock this issue now, thanks for the entertainment, people pay real money for this shit, and we got it for free 💵

:salt: :joy: u aint shit mate
